assets/src/components/editing/elements/common/AudioClipPicker.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useRef } from 'react';
```

Comment:
Refactored out the audio pick from the pronunciation component since it's also can be used in the <tc> components now.